jvbarisone89/day4hw/events_lab

```diff
@@ -0,0 +1 @@
+Subproject commit 756e605c2b3acf823e2d12fdd190a6c4dd85a7db
```

Comment:
This often indicates that you copied another github repository under your 23-modules repository. Unfortunately that won't work. If this is a lab that you were asked to fork, please remove it from here and put it in your `dev/` directory. Then push to the fork and make a pull-request on **that lab's repo**

If this is just a few files you can instead remove the hidden .git directory from 23-homework/jsnow/some_lab/.git (warning don't remove 23-modules/.git)